recipes/mongo-cxx-driver/all/conanfile.py

```diff
@@ -13,6 +13,7 @@
 class MongoCxxConan(ConanFile):
     name = "mongo-cxx-driver"
     license = "Apache-2.0"
+    version = "3.8.1"
```

Comment:
```suggestion
```
Please, do not include the package version as recipe attribute. https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#version